lib/Driver/Driver.cpp

```diff
@@ -672,6 +672,88 @@ getDriverBatchCount(llvm::opt::InputArgList &ArgList,
   return None;
 }
 
+static bool computeIncremental(const llvm::opt::InputArgList *ArgList,
+                               const bool ShowIncrementalBuildDecisions) {
+  {
+    const bool WasIncrementalRequested =
+    ArgList->hasArg(options::OPT_incremental);
```

Comment:
Indentation got messed up here, but also it seems like this doesn't need its own variable at all.

Reply:
I tried it without the variable and you are right; it's better that way.